### PR TITLE
Added New FlipImage Module

### PR DESCRIPTION
**Pull request by HarshKhandeparkar**
Fixes #[Add issue number here.]

This new module flips the image on the selected axis.

flip-img-module

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `npm test`
* [x] code is in uniquely-named feature branch and has no merge conflicts
* [x] PR is descriptively titled
* [x] ask `@publiclab/is-reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software

Thanks!

### CODE REVIEW

**Comment by aashna27** [comment]
This is some really good work !! 🙌

File: src/modules/FlipImage/Module.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
+/*
+ * Display only one color channel
+ */
+module.exports = function Channel(options, UI) {
```
Comment (Mridul97) [inline src/modules/FlipImage/Module.js]: Great Work! Could you please change the comments to something more appropriate and also changing the function name would be nice!

Comment (HarshKhandeparkar) [reply src/modules/FlipImage/Module.js]: Oh I had copied the boilerplate from channel. will change that soon 😆😆

Comment (HarshKhandeparkar) [reply src/modules/FlipImage/Module.js]: Changes Done!

Comment (HarshKhandeparkar) [reply src/modules/FlipImage/Module.js]: Will add the dist later.

Comment (HarshKhandeparkar) [reply src/modules/FlipImage/Module.js]: Please review the current changes

**Comment by HarshKhandeparkar** [comment]
@publiclab/is-reviewers I have rebased. Please review.

**Comment by HarshKhandeparkar** [comment]
@jywarren @publiclab/is-reviewers conflicts fixed!!

**Comment by jywarren** [comment]
Hi, i tried a couple times to rebase this but got some conflicts. Perhaps you should try squashing the commits first and then rebasing? Oddly the conflicts were in the UI code, which you haven't touched here. Can you give it a try?

**Comment by HarshKhandeparkar** [comment]
@jywarren @publiclab/is-reviewers Merged with main!!

**Comment by HarshKhandeparkar** [comment]
@jywarren can you please merge my PRs before the other ones since I will not be able to touch my PC until 16th of april. I don't know if I will get time past that date as well.

**Comment by jywarren** [comment]
Whew, that was a tough rebase. Let's see how it does.

**Comment by jywarren** [comment]
🎉

**Comment by HarshKhandeparkar** [comment]
Thanks!!!

**Pull request merged**